internal/exec/stages/files/files.go

```diff
@@ -234,7 +234,15 @@ func (lst ByDirectorySegments) Swap(i, j int) {
 }
 
 func (lst ByDirectorySegments) Less(i, j int) bool {
-	return lst[i].Depth() < lst[j].Depth()
+	return depth(lst[i].Node) < depth(lst[j].Node)
+}
+
+func depth(n types.Node) int {
```

Comment:
The reason I made this an `int` is because [the function this is copied from](https://github.com/coreos/ignition/blob/eb48ac9a69ce2c3fe97d9b9bea5aae1c252d6911/config/types/node.go#L51) has this as an `int`.

If this ever gets large enough to reach the limits of an `int` we've probably got larger problems, but `uint` does make more sense for this function. I'll switch it to use `uint`.